### PR TITLE
fix: corrected bad variable translations in pt_PT

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Fix a bad translation that was causing errors to our push/pull translation jobs

JIRA: [EDUCATOR-5750](https://openedx.atlassian.net/browse/EDUCATOR-5750)

**What changed?**

- Variables for injection into a translated string were accidentally modified. This fixes those variable injections in the pt_PT translation files.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

Locally:
- In devstack with local ORA installed and in the `edx-ora2` virtual environment, run `make check_translations_up_to_date`. 
- Previously, this would fail. Verify that the job completes (with error is fine, you should see a `Source translations are out-of-date! Please update them.` error which is expected).

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by nsprenkle** [comment]
Closing since @sarina was able to fix in transifex
